shared/common-adapters/icon.native.js

```diff
@@ -23,7 +23,8 @@ class Icon extends Component<void, Exact<Props>, void> {
     const width = this.props.style && this.props.style.width && {width: this.props.style.width}
     const height = this.props.style && this.props.style.height && {height: this.props.style.height}
 
-    const fontSize = this.props.style && (this.props.style.fontSize || this.props.style.width)
+    const fontSizeHint = shared.fontSize(iconType)
+    const fontSize = (this.props.style && (this.props.style.fontSize || this.props.style.width) && {fontSize: this.props.style.fontSize || this.props.style.width}) || fontSizeHint
```

Comment:
:sheep: Perhaps extract `this.props.style.fontSize || this.props.style.width` to a named const above?